.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+    "es2015",
```

Comment:
Using es6+ here will prevent us from using this in any repo that hasn't been converted yet, unless we're going to compile the components down to es5 and then import them. Not necessarily bad but something to keep in mind.